javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -248,6 +238,315 @@ public boolean isOperator() {
         }
     }
 
+    @Generated("com.github.javaparser.generator.core.other.TokenKindGenerator")
+    public enum Kind {
+
+        EOF(0), SPACE(1), WINDOWS_EOL(2), UNIX_EOL(3), OLD_MAC_EOL(4), SINGLE_LINE_COMMENT(5), ENTER_JAVADOC_COMMENT(6), ENTER_MULTILINE_COMMENT(7), JAVADOC_COMMENT(8), MULTI_LINE_COMMENT(9), COMMENT_CONTENT(10), ABSTRACT(11), ASSERT(12), BOOLEAN(13), BREAK(14), BYTE(15), CASE(16), CATCH(17), CHAR(18), CLASS(19), CONST(20), CONTINUE(21), _DEFAULT(22), DO(23), DOUBLE(24), ELSE(25), ENUM(26), EXTENDS(27), FALSE(28), FINAL(29), FINALLY(30), FLOAT(31), FOR(32), GOTO(33), IF(34), IMPLEMENTS(35), IMPORT(36), INSTANCEOF(37), INT(38), INTERFACE(39), LONG(40), NATIVE(41), NEW(42), NULL(43), PACKAGE(44), PRIVATE(45), PROTECTED(46), PUBLIC(47), RETURN(48), SHORT(49), STATIC(50), STRICTFP(51), SUPER(52), SWITCH(53), SYNCHRONIZED(54), THIS(55), THROW(56), THROWS(57), TRANSIENT(58), TRUE(59), TRY(60), VOID(61), VOLATILE(62), WHILE(63), REQUIRES(64), TO(65), WITH(66), OPEN(67), OPENS(68), USES(69), MODULE(70), EXPORTS(71), PROVIDES(72), TRANSITIVE(73), LONG_LITERAL(74), INTEGER_LITERAL(75), DECIMAL_LITERAL(76), HEX_LITERAL(77), OCTAL_LITERAL(78), BINARY_LITERAL(79), FLOATING_POINT_LITERAL(80), DECIMAL_FLOATING_POINT_LITERAL(81), DECIMAL_EXPONENT(82), HEXADECIMAL_FLOATING_POINT_LITERAL(83), HEXADECIMAL_EXPONENT(84), HEX_DIGITS(85), UNICODE_ESCAPE(86), CHARACTER_LITERAL(87), STRING_LITERAL(88), IDENTIFIER(89), LETTER(90), PART_LETTER(91), LPAREN(92), RPAREN(93), LBRACE(94), RBRACE(95), LBRACKET(96), RBRACKET(97), SEMICOLON(98), COMMA(99), DOT(100), AT(101), ASSIGN(102), LT(103), BANG(104), TILDE(105), HOOK(106), COLON(107), EQ(108), LE(109), GE(110), NE(111), SC_OR(112), SC_AND(113), INCR(114), DECR(115), PLUS(116), MINUS(117), STAR(118), SLASH(119), BIT_AND(120), BIT_OR(121), XOR(122), REM(123), LSHIFT(124), PLUSASSIGN(125), MINUSASSIGN(126), STARASSIGN(127), SLASHASSIGN(128), ANDASSIGN(129), ORASSIGN(130), XORASSIGN(131), REMASSIGN(132), LSHIFTASSIGN(133), RSIGNEDSHIFTASSIGN(134), RUNSIGNEDSHIFTASSIGN(135), ELLIPSIS(136), ARROW(137), DOUBLECOLON(138), RUNSIGNEDSHIFT(139), RSIGNEDSHIFT(140), GT(141), CTRL_Z(142);
```

Comment:
Could we generate it with one per line?

Reply:
Everything gets reformatted because this file is now part of the code generation step. See also #1301 :-)